appwrite/website
# Add blog post on apple sign in

## What does this PR do?

- Adds blog post on sign in with Apple / Appwrite
- Add blog post assets

## Test Plan

- Run the website and navigate to `/blog/post/how-to-set-up-sign-in-with-apple`
- Confirm that blog post exists and that the assets are displayed correctly

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes
